test/blackbox/blackbox.spec.ts

```diff
@@ -33,6 +33,7 @@ const AsyncAPIV2_3Files = readFilesInFolder('AsyncAPI-2_3');
 const AsyncAPIV2_4Files = readFilesInFolder('AsyncAPI-2_4');
 
 const filesToTest = [
+<<<<<<< HEAD
```

Comment:
You have some merge conflicts here that has not been solved 🙂

Reply:
Whoops, sorry about that! My editor usually raises a five-alarm fire if I miss merge conflicts, so I've gotten sloppy. :joy: Blocking out time to fix + retest.